cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpLoggingInterceptor.java

```diff
@@ -37,6 +42,17 @@ void redactHeader(String header) {
         DELEGATE.redactHeader(header);
```

Comment:
the `void level(HttpLoggingIntercepter.Level level)` is now confusing with the `setLevel()` and `setLogLevel` methods. It's also unused

Reply:
removed the unused level method